src/downshift.js

```diff
@@ -679,9 +686,15 @@ class Downshift extends Component {
   }
 
   input_handleBlur = () => {
-    if (!this.isMouseDown) {
-      this.reset({type: Downshift.stateChangeTypes.blurInput})
-    }
+    // Need setTimeout, so that when the user presses Tab, the activeElement is the next focused element, not the body element
+    setTimeout(() => {
+      const downshiftButtonIsActive =
+        this.props.environment.document.activeElement.tagName === 'BUTTON' &&
+        this.props.environment.document.activeElement.dataset.toggle
```

Comment:
I actually preferred when this was set to `contains`, I'd rather change this to be:

```javascript
const downshiftButtonIsActive =
  this.props.environment.document.activeElement.dataset.toggle &&
  this._rootNode.contains(this.props.environment.document.activeElement)
```

Reply:
Oh, yeah that makes a lot more sense